code/game/objects/items/plushes.dm

```diff
@@ -118,9 +120,24 @@
 			if(!stuffed)
 				to_chat(user, "<span class='warning'>You already murdered it!</span>")
 				return
-			user.visible_message("<span class='notice'>[user] tears out the stuffing from [src]!</span>", "<span class='notice'>You rip a bunch of the stuffing from [src]. Murderer.</span>")
-			I.play_tool_sound(src)
-			stuffed = FALSE
+			if(!divine)
+				user.visible_message("<span class='notice'>[user] tears out the stuffing from [src]!</span>", "<span class='notice'>You rip a bunch of the stuffing from [src]. Murderer.</span>")
+				I.play_tool_sound(src)
+				stuffed = FALSE
+			else
+				to_chat(user, "<span class='notice'>What a fool you are. [src] is a god, how can you kill a god? What a grand and intoxicating innocence.</span>")
+				if(iscarbon(user))
+					var/mob/living/carbon/C = user
+					if(C.drunkenness < 50)
+						C.drunkenness = min(C.drunkenness + 20, 50)
+				var/turf/current_location = get_turf(user)
+				var/area/current_area = current_location.loc //copied from hand tele code
+				if(current_location && current_area && current_area.noteleport)
+					to_chat(user, "<span class='notice'>There is no escape. No recall or intervention can work in this place.</span>")
+				else
+					to_chat(user, "<span class='notice'>There is no escape. Although recall or intervention can work in this place, attempting to flee from [src]'s immense power would be futile.</span>")
```

Comment:
This seems oddly specific to me, just sounds a bit weird with the whoole 'There is not escape. Although technically you could but you shouldn't' thing

Reply:
It's a pretty cool morrowind reference